### PR TITLE
feat: add HAQQ Network Mainnet

### PR DESCRIPTION
**Pull request by kioqq**

### CODE REVIEW

File: haqq/chain.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "haqq",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://haqq.network/",
+  "pretty_name": "Haqq Network",
+  "chain_id": "haqq_11235-1",
+  "bech32_prefix": "haqq",
```
Comment (JeremyParish69) [inline haqq/chain.json:9]: Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: haqq/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "ISLM",
+          "exponent": 18
+        }
+      ],
+      "base": "aISLM",
+      "name": "Islamic Coin",
+      "display": "islm",
```
Comment (JeremyParish69) [inline haqq/assetlist.json:19]: the display denom must be listed in the denom_units array and is case sensitive. Please either correct display to ISLM (make uppercase), or the latter denom_unit to islm (to lowercase)

File: haqq/chain.json (hunk 1 above)
Comment (kioqq) [reply haqq/chain.json:9]: https://github.com/satoshilabs/slips/pull/1576 ✅

File: haqq/assetlist.json (hunk 2 above)
Comment (kioqq) [reply haqq/assetlist.json]: Fixed, thank you ❤️

**Pull request merged**